javascript-data-model/order-history.js

```diff
@@ -0,0 +1,81 @@
+var orderHistory = [
+  {
+    placed: 'August 4, 2020',
+    total: 34.00,
+    to: 'JS Masher',
+    order: '114-3941689-8772232',
+    details: '',
+    invoice: '',
+    delivered: 'Aug 8, 2020',
+    item: [
+      {
+        image: [''],
+        title: ['Javascript for impatient programmers'],
+        author: ['Rauschmayer, Dr. Axel'],
+        return: ['Return window closed on Sep 7, 2020'],
+        price: [31.55]
```

Comment:
There's no need for these to be arrays now that you have revised your data model for this list of items to be an array of objects. Each object should hold the data for just that one object, so there's no need to prepare for lists here. These can be stored as just the strings and numbers directly.